python/cudf/cudf/core/reshape.py

```diff
@@ -1160,3 +1160,257 @@ def _length_check_params(obj, columns, name):
                 f"length of the columns being "
                 f"encoded ({len(columns)})."
             )
+
+
+def _get_names(arrs, names, prefix: str = "row"):
```

Comment:
Can we add a one or two sentence description of what this function does? The name is unfortunately not descriptive enough..

Reply:
Added the description.